.github/workflows/proselint.yml

```diff
@@ -0,0 +1,37 @@
+name: proselint
+
+on:
+  # Trigger the workflow on push or pull requests, but only for the
+  # master branch
+  push:
+    branches:
+    - master
+    - '*/ci'
+  pull_request:
+    branches:
+    - master
+
+jobs:
+  check:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: install prereqs
+      run: sudo apt-get install python3-proselint
+
+    # config file help: https://github.com/amperser/proselint/
+    - name: store proselint config
+      run: |
+        echo '{' > $HOME/.proselintrc
```

Comment:
This seems a weird way of doing this, why not store it under the CI folder then copy it into place?

Reply:
sounds like a better approach indeed, I'll try that!